onnxruntime/python/tools/quantization/E2E_example_model/e2e_user_bert_example.py

```diff
@@ -0,0 +1,21 @@
+import os
+from onnxruntime.quantization import BertDataReader, generate_calibration_table
+
+def get_calibration_table(model_path, augmented_model_path):
+    data_reader = BertDataReader(model_path)
+    generate_calibration_table(model_path, augmented_model_path, data_reader)
+
+def get_prediction_evaluation(model_path):
+    data_reader = BertDataReader(model_path)
+    data_reader.evaluate()
+
+
+if __name__ == '__main__':
+
+    model_path = 'bert.shape.onnx'
+    augmented_model_path = 'augmented_bert.shape.onnx'
+
+    get_calibration_table(model_path, augmented_model_path)
+    get_prediction_evaluation(model_path)
```

Comment:
Is this BERT example working? Should we wait until BERT INT8 perf meets the expectation before we merge it?

Reply:
BERT example is working, however I agree to not to add it until BERT INT8 perf meets the expectation. So I removed it.